README.rst

```diff
@@ -678,6 +678,25 @@ information by performing lookups on the OUI
 
 This feature is enabled by default.
 
+``OPENWISP_MONITORING_RETRY_OPTIONS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+--------------------------------------------------------------------------------------------------+
+| **type**:    |                                            ``dict``                                              |
```

Comment:
please do not do this thing of aligning the text to the center because it will be hard to maintain (plus not useful)